package-lock.json

```diff
@@ -0,0 +1,2754 @@
+{
+  "name": "node-tailor",
```

Comment:
we use `yarn.lock`, can you update yarn.lock instead?

Reply:
Yes, will do!

Reply:
There were no updates, I just happened to use latset npm & node %)

Reply:
Done